napari/_qt/qt_viewer.py

```diff
@@ -270,26 +270,15 @@ def __init__(
         )
         self.canvas.events.draw.connect(self.camera.on_draw)
 
-        # Create the experimental QtPool for octree and/or monitor.
+        # Create the experimental QtPool for monitor if necessary.
         self._qt_poll = _create_qt_poll(self, self.viewer.camera)
 
         # Create the experimental RemoteManager for the monitor.
         self._remote_manager = _create_remote_manager(
             self.viewer.layers, self._qt_poll
         )
 
-        # moved from the old layerlist... still feels misplaced.
-        # can you help me move this elsewhere?
-        if config.async_loading:
-            from napari._qt.experimental.qt_chunk_receiver import (
-                QtChunkReceiver,
-            )
-
-            # The QtChunkReceiver object allows the ChunkLoader to pass newly
-            # loaded chunks to the layers that requested them.
-            self.chunk_receiver = QtChunkReceiver(self.layers)
-        else:
-            self.chunk_receiver = None
+        self.chunk_receiver = None
```

Comment:
Could we just remove this attribute?

Or are you trying to keep backwards compatibility here? If so, I think making this a property that issues a deprecation warning and returns `None` is preferred.

Reply:
I wasn't trying to preserve backwards compatibility (that seems like a nightmare here...). This was a leftover from soemthing else I was trying do and needs to be cleaned up :)